make-pdf.rb

```diff
@@ -39,19 +54,33 @@
 box_w = PAGE_WIDTH.mm / n_cols
 box_h = PAGE_HEIGHT.mm / n_rows
 
-Prawn::Document.generate(options.output || 'postcard.pdf') do
-  curr_color = options.base_color
+def get_hex_color(color)
+  if COLORS[color] != nil
+    # User specified color name, so return hex
+    return COLORS[color]
+  elsif !color[/\H/] && color.length == 6
+    # User specified hex
+    return color
+  else
+    # Something's not right
+    puts "Invalid color specified."
+    exit
```

Comment:
I guess that we never get into this code path, since `color` is either in `COLORS` or not

Reply:
Oh, sorry, another if is more complex, but I'm still not sure that `color.length == 6` is good here. I didn't check, but probable `prawn` accepts also short hex color names, e.g. `#fff` or `#111` like in CSS, does it?

Reply:
Ok, I fixed the quotes. Not sure how to configure rufo to do that automatically, so I just did find and replace. 😃 

Good point, I didn't think about the short hex colors. I just tried a few of them, and it throws an error that says "unknown type of color". So it apparently does not support it.